### PR TITLE
[KO] Update `Support Team` to `Technical Support Team`

### PR DESCRIPTION
**Pull request by Sitoria**
## Check

- [x] #10425
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] @CivilWiki 
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by CivilWiki — APPROVED** [review]
all good

**Comment by agatemosu** [comment]
rename `meta/group-info` too

**Comment by Sitoria** [comment]
> [update](https://osu.wiki/status/diff/0522bc4c323f9a2b0c89a6b9582109a0608c6287/meta/group-info/en.yaml) `meta/group-info` too

ko doesn't have yaml file 😰

**Comment by agatemosu** [comment]
https://github.com/Sitoria/osu-wiki/blob/Team/meta/group-info/ko.yaml

I see the yaml file 😰

**Comment by Sitoria** [comment]
> https://github.com/Sitoria/osu-wiki/blob/Team/meta/group-info/ko.yamlt
> I see the yaml file 😰

its too outdated - will inv later

File: wiki/Main_page/ko.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ osu!와 커뮤니티가 음악 라이센싱과 미디어 사용 가이드라인
 
 많은 유저들이 개발, 유지보수 및 커뮤니티 관리에 도움을 주지 않았더라면 osu!는 불가능했을 것입니다. 그들의 노력, 시간, 그리고 헌신이 osu!를 더욱더 활기차고 발전시킬 수 있게 만들어줍니다.
 
-[osu! 팀](/wiki/People/osu!_team) • [개발자](/wiki/People/Developers) • [공식 아티스트](/wiki/People/Featured_Artists) • [글로벌 모더레이션 팀](/wiki/People/Global_Moderation_Team) • [지원 팀](/wiki/People/Support_Team) • [노미네이션 어세스먼트 팀](/wiki/People/Nomination_Assessment_Team) • [비트맵 노미네이터](/wiki/People/Beatmap_Nominators) • [osu! Alumni](/wiki/People/osu!_Alumni) • [Project Loved 팀](/wiki/People/Project_Loved_Team) • [비트맵 스포트라이트 큐레이터](/wiki/People/Beatmap_Spotlight_Curators)
+[osu! 팀](/wiki/People/osu!_team) • [개발자](/wiki/People/Developers) • [공식 아티스트](/wiki/People/Featured_Artists) • [글로벌 모더레이션 팀](/wiki/People/Global_Moderation_Team) • [기술 지원 팀](/wiki/People/Technical_Support_Team) • [노미네이션 어세스먼트 팀](/wiki/People/Nomination_Assessment_Team) • [비트맵 노미네이터](/wiki/People/Beatmap_Nominators) • [osu! Alumni](/wiki/People/osu!_Alumni) • [Project Loved 팀](/wiki/People/Project_Loved_Team) • [비트맵 스포트라이트 큐레이터](/wiki/People/Beatmap_Spotlight_Curators)
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Main_page/ko.md:114]: 그럼 `지원팀`으로 맞출게요

**Comment by Sitoria** [comment]
~~2 nominated so can approved~~

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**